src/Features/Core/Portable/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -73,8 +74,6 @@ public IStreamingProgressTracker ProgressTracker
         // any of these.
         public ValueTask OnStartedAsync(CancellationToken cancellationToken) => default;
         public ValueTask OnCompletedAsync(CancellationToken cancellationToken) => default;
-        public ValueTask OnFindInDocumentStartedAsync(Document document, CancellationToken cancellationToken) => default;
-        public ValueTask OnFindInDocumentCompletedAsync(Document document, CancellationToken cancellationToken) => default;
```

Comment:
FAR had a system to notify consumers "i'm startign a doc" and "i'm done with a doc". The original thinking was to use that for progress, to display which file we're looking at.

but:

we never hooked this up to naything at all.
this wouldn't be a good experience anyways, since we're extremely parallel. so this was removed entirely.